src/webgl/material.js

```diff
@@ -178,7 +178,7 @@ p5.prototype.ambientMaterial = function(v1, v2, v3, a) {
   gl.useProgram(shaderProgram);
   shaderProgram.uMaterialColor = gl.getUniformLocation(
     shaderProgram, 'uMaterialColor' );
-  var colors = this._renderer._applyColorBlend(v1,v2,v3,a);
+  var colors = this._renderer._applyColorBlend.apply(this._renderer, arguments);
```

Comment:
I'm not sure if I see where the bug is here.  apply is called on the current renderer context within _applyColorBlend.  If we want to call apply outside of the _applyColorBlend function, we should rename _applyColorBlend to _blendColor and tweak that function so as not to apply separate context inside.